### PR TITLE
Add 'eta' alias to EJS

### PR DESCRIPTION
**Pull request by nebrelbug**
[Eta](https://eta.js.org) is fairly similar to EJS. If it ever gets popular, I might throw together a new pull request with proper support for strings and comments inside delimiters :)

### CODE REVIEW

File: components.json [hunk 1]
```diff
@@ -291,7 +291,8 @@
 		"ejs": {
 			"title": "EJS",
 			"require": ["javascript", "markup-templating"],
-			"owner": "RunDevelopment"
+			"owner": "RunDevelopment",
+			"alias": "eta"
```
Comment (RunDevelopment) [inline components.json:295]: You should also give this an `aliasTitle`, similar to how `svg` is an alias of `markup` with the title `SVG`.

Comment (nebrelbug) [reply components.json]: Done :)

File: plugins/show-language/prism-show-language.js [hunk 2]
```diff
@@ -57,6 +57,7 @@
 		"dockerfile": "Docker",
 		"ebnf": "Extended Backus–Naur form",
 		"ejs": "EJS",
+		"eta": "EJS",
```
Comment (RunDevelopment) [inline plugins/show-language/prism-show-language.js:60]: This isn't good. It should be `ETA`.
That's a bug in Prism's build. I'll investigate.

Comment (RunDevelopment) [reply plugins/show-language/prism-show-language.js:60]: Will be fixed by #2283. Let's merge this PR!

**Comment by RunDevelopment** [comment]
Thank you for contributing @nebrelbug!

**Comment by nebrelbug** [comment]
@RunDevelopment thank you!

**Pull request merged**